app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/CheckContactDetailsMrnController.scala

```diff
@@ -101,6 +101,9 @@ class CheckContactDetailsMrnController @Inject() (
   def submit(implicit journey: JourneyBindable): Action[AnyContent] =
     authenticatedActionWithSessionData.async { implicit request =>
       withAnswersAndRoutes[MrnContactDetails] { (fillingOutClaim, _, router) =>
+        implicit val routes: ReimbursementRoutes = extractRoutes(fillingOutClaim.draftClaim, journey)
```

Comment:
you don't need to have this implicit val here  or the import - you can use 'router'. 
withAnswersAndRoutes extracts the routes and journey.

Reply:
@shivalasvegas I've removed it